linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_rename/rule.yml

```diff
@@ -49,23 +63,31 @@ references:
     srg: SRG-OS-000064-GPOS-00033,SRG-OS-000392-GPOS-00172,SRG-OS-000458-GPOS-00203,SRG-OS-000461-GPOS-00205,SRG-OS-000468-GPOS-00212
     stigid@sle15: SLES-15-030710
     vmmsrg: SRG-OS-000458-VMM-001810,SRG-OS-000461-VMM-001830
-
-{{{ complete_ocil_entry_audit_syscall(syscall="rename") }}}
-
 warnings:
     - general: |-
         Note that these rules can be configured in a
         number of ways while still achieving the desired effect. Here the system calls
         have been placed independent of other system calls. Grouping system calls related
         to the same event is more efficient. See the following example:
+        {{% if product not in ["sle15"] %}}
         <pre>-a always,exit -F arch=b32 -S unlink,unlinkat,rename,renameat -F exit=-EACCES -F auid>={{{ auid }}} -F auid!=unset -F key=unsuccesful-delete</pre>
+        {{% else %}}
+        <pre>
+        -a always,exit -F arch=b32 -S rename -F auid>=1000 -F auid!=4294967295 -k perm_mod
+        -a always,exit -F arch=b64 -S rename -F auid>=1000 -F auid!=4294967295 -k perm_mod
+        </pre>
+        {{% endif %}}    
 
+
+{{{ complete_ocil_entry_audit_syscall(syscall="rename") }}}
 template:
+    {{% if product not in ["sle15"] %}}
     name: audit_rules_unsuccessful_file_modification
     vars:
         name: rename
-        syscall_grouping:
-          - rename
-          - renameat
-          - unlink
-          - unlinkat
+    {{% else %}}
+    name: audit_rules_syscal_events
```

Comment:
Why has `syscall_grouping` been removed?

Reply:
Hello @jan-cerny 

With this correction I am trying to fix the issues described here
here https://github.com/ComplianceAsCode/content/issues/7387#issuecomment-903749625

Because of that I have decided to use an template. When I did a PR I received a notification about rebase – for example you can check this PR which is also mine
https://github.com/ComplianceAsCode/content/pull/7487 – check the rule.yml

The main reason for this need of rebase is coming from the fact that in the past 
we had a dedicated rule for each system call – for example rename, renameat, ulink, unlinkat and now we have a new design of the rule which includes section syscall_grouping.

May you help me with these reabase issues, because I think only the part of the rule between 
{{% if product not in ["sle15"] %}} and  {{% else %}}
does not allow me to continue. For example I used this part of code in PR 7487 and I had a request for a rebase

template:
    {{% if product not in ["sle15"] %}}
    name: audit_rules_unsuccessful_file_modification
    vars:
        name: renameat
        syscall_grouping:
          - rename
          - renameat
          - unlink
          - unlinkat
 
        {{% else %}}
    name: audit_rules_syscal_events
    vars:
        attr: renameat
    {{% endif %}}
 

Thank you in advance for your help!!